### DISCUSSION TITLE
How to prevent seeking until certain percentage reached?

### DISCUSSION THREAD

**Discussion by shanebeehler** [vidstack/player#1079] [Q&A] [answered]
I want to be able to prevent the user from seeking past where they've watched until they've watched, for example, 90% of the video. I tried the `disabled` prop on `TimeSlider.Root` but I need to still be able to show the `TimeSlider.Preview` to show a tooltip explaining why seeking is disabled

**Comment by mihar-22** [comment, marked as answer]
This is possible by stopping the `media-seek-request` event dispatched by the `TimeSlider` but... we haven't exposed the event callback and types for it on the slider. I've noted it down as a feature request for the next release.

**Reply by shanebeehler** [reply]
Thank you!
